tools/nimgrep.nim

```diff
@@ -115,20 +157,23 @@ type
                     lineBeg: int, lineEnd: int, match: string]
   outputKind = enum
     OpenError, Rejected, JustCount,
-    BlockFirstMatch, BlockNextMatch, BlockEnd, FileContents
+    BlockFirstMatch, BlockNextMatch, BlockEnd, FileContents, FileName
```

Comment:
leave as a TODO for future PR but style is wrong, should be: `OutputKind = enum openError, ...`

Reply:
done in this PR, thank you!